pstress/pstress-run.sh

```diff
@@ -8,7 +8,6 @@
 CONFIGURATION_FILE=pstress-run.conf  # Do not use any path specifiers, the .conf file should be in the same path as pstress-run.sh
 
 # ========================================= Improvement ideas ====================================================================
-# * SAVE_TRIALS_WITH_CORE_OR_VALGRIND_ONLY=0 (These likely include some of the 'SIGKILL' issues - no core but terminated)
```

Comment:
Why this variable is removed. This variable is used when there are CORES found. You should have renamed the variable to SAVE_TRIALS_WITH_CORE_ONLY